SU2_CFD/include/numerics_structure.hpp

```diff
@@ -66,13 +66,13 @@ class CNumerics {
   su2double *Enthalpy_formation;
   su2double Prandtl_Lam;        /*!< \brief Laminar Prandtl's number. */
   su2double Prandtl_Turb;    /*!< \brief Turbulent Prandtl's number. */
-  
```

Comment:
I don't think that this is intended. Usually in some editors the default is to remove empty lines when saving the file. In fact, I would suggest to revert these changes as they give the potential to lead to conflicts in the future. We should be consistent regarding the handling of white spaces and empty lines.